wallet/src/ui/app/components/network-switch/NetworkSelector.tsx

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import cl from 'classnames';
+import { useMemo, useCallback } from 'react';
+
+import { API_ENV_TO_INFO, API_ENV } from '_app/ApiProvider';
+import BsIcon from '_components/bs-icon';
+import { useAppSelector, useAppDispatch } from '_hooks';
+import {
+    setApiEnv,
+    changeRPCNetwork,
+    setNetworkSelector,
+} from '_redux/slices/app';
+import { getTransactionsByAddress } from '_redux/slices/txresults';
+import store from '_store';
+
+import st from './Network.module.scss';
+
+const NetworkSelector = () => {
+    const selectedApiEnv = useAppSelector(({ app }) => app.apiEnv);
+    const dispatch = useAppDispatch();
+    const networkList: any = API_ENV_TO_INFO;
```

Comment:
I think `any` is not allowed by our linting rules, and in general if possible we should avoid it.